fecgov/openFEC
# 4892- Add committee_type filter and test

## Summary (required)

- Resolves #4892 

Adds the `committee_type` filter to gradually change `type` to `committee_type` per our deprecation policy. Also, updates some test names to match with the `entity_type` change. Also, adds a test for the new filter (thanks @pkfec!)

### Required reviewers

1-2 devs

## Impacted areas of the application

General components of the application that this PR will affect:

-  financials/reports/{entity_type} endpoint 

## Screenshots
<img width="1049" alt="Screen Shot 2022-02-03 at 2 53 46 PM" src="https://user-images.githubusercontent.com/66386084/152419636-9d774d39-6e9b-4e71-a7f2-3c9c54bd45dd.png">



## How to test

- Check out this branch
- Try queries for financials/reports/<string:entity_type>/    using the old `type` and new `committee_type` filters, should work with both, either, or neither!
- http://localhost:5000/v1/reports/pac-party/?sort_nulls_last=false&sort=-committee_id&sort_null_only=false&per_page=20&sort_hide_null=false&api_key=DEMO_KEY&cycle=2022&page=1
